### ISSUE TITLE
[Community event] Translate documentation to your own langage

### ISSUE THREAD

**Issue by Wauplin** [huggingface/huggingface_hub#1700]
`huggingface_hub` documentation is currently translated from English to German and Korean. Which language will be next? This [translation guide](https://github.com/huggingface/huggingface_hub/blob/main/docs/TRANSLATING.md) describes how to get started with a new one.

**Current list:**
- [ ] French: https://github.com/huggingface/huggingface_hub/issues/1897
- [x] German: https://github.com/huggingface/huggingface_hub/issues/1645
- [ ] Hindi: https://github.com/huggingface/huggingface_hub/issues/1771
- [ ] Korean: https://github.com/huggingface/huggingface_hub/issues/1626
- [ ] Russian: https://github.com/huggingface/huggingface_hub/issues/3114
- [ ] Simplified Chinese: https://github.com/huggingface/huggingface_hub/issues/1915
- [ ] Spanish: https://github.com/huggingface/huggingface_hub/issues/2640
- [ ] Tamil: https://github.com/huggingface/huggingface_hub/issues/1901
- [ ] Urdu: https://github.com/huggingface/huggingface_hub/issues/1820

**Comment by mohsinmahmood12** [comment]
@Wauplin I want to work on this issue. Translating the docs to Urdu Language

**Comment by Wauplin** [comment]
Hi @mohsinmahmood12, you are more than welcome to start translating docs to Urdu! Thanks for proposing your help on this! :pray: I created an issue dedicated to it: https://github.com/huggingface/huggingface_hub/issues/1820. What I would suggest is to first bootstrap the Urdu docs by translating the README and quickstart guides (similar to this PR for Hindi https://github.com/huggingface/huggingface_hub/issues/1820). Once that's done, you'll have the project structure ready to translate other guide one by one -you or any other Urdu-speaking community member- :hugs: 

cc @abidlabs who will like this :wink:

**Comment by AnushSomasundaram** [comment]
@Wauplin I can help translate the documentation into Tamil.

**Comment by Wauplin** [comment]
@AnushSomasundaram Nice! Thank you for proposing your contribution on this! What I suggest you to do first is to create an issue dedicated to Tamil [from this link](https://github.com/huggingface/huggingface_hub/issues/new?assignees=&labels=WIP&projects=&template=i18n.md&title=%5Bi18n-%3ClanguageCode%3E%5D+Translating+docs+to+%3ClanguageName%3E). Then it's good practice to start with a small PR to bootstrap the project with a translated README + quickstart guide for example (see [Hindi example](https://github.com/huggingface/huggingface_hub/pull/1772/files)). That way documentation will be generated automatically and anyone can contribute by adding new pages one PR at a time. What do you think? :)

**Comment by AnushSomasundaram** [comment]
Works. Will get on it.

**Comment by Raghul-M** [comment]
@Wauplin Is this still open ?  Translating the docs to Tamil Language

**Comment by Wauplin** [comment]
Hi @Raghul-M, thanks for proposing yourself! Yes this task is an ongoing work and Tamil and not yet been started. Please refer to https://github.com/huggingface/huggingface_hub/issues/1901 regarding Tamil translation. You will find instructions on how to start. The best would be to start with a small PR to see answer all the questions you can have and then continue with the rest. Also note that you'll need to encourage other Tamil-speaking contributors to start the journey with you as I'll not be able to review the translation themselves. :hugs:

**Comment by la21unika** [comment]
@Wauplin I am a native Spanish speaker. Although not on the list, if there is a need to translate any documents, I would like to volunteer. 
Thanks,

**Comment by Wauplin** [comment]
Hi @la21unika, thanks for proposing yourself! :hugs:
As I mentioned above, this is an ongoing work where anyone can contribute in their own language. I just created a new issue for the Spanish-speaking community! https://github.com/huggingface/huggingface_hub/issues/2640 First thing to do would be to start with a small PR to start getting a structure for the docs in Spanish (following the [translation guide](https://github.com/huggingface/huggingface_hub/blob/main/docs/TRANSLATING.md)). Here is a recent example in Tamil: https://github.com/huggingface/huggingface_hub/pull/2555 if you want to have a look. Once the PR is opened, we recommend to ask in the Spanish-speaking community if anyone wants to review your work. If you don't find anyone, please let us know and we will find a solution. Let me know if you have any questions!

**Comment by vaibhav-research** [comment]
Hi @Wauplin 

I am new to this community and looking for a project to start my contribution. Is there any open issue left that I can pick ?

**Comment by Wauplin** [comment]
Hi @vaibhav-research , would you like to contribute to translate documentation as suggested in this issue? Which language could you pick?

**Comment by vaibhav-research** [comment]
Thanks for your response @Wauplin , I was looking for Hindi but I saw that PR is already merged. I can take a shot at anything else which is available. I will get a chance to explore other language in the process

**Comment by DiyorbekDil** [comment]
Hi! I’d like to help by translating the documentation to Uzbek. Is this still available?

**Comment by Wauplin** [comment]
@DiyorbekDil  Yes it is! See [translating guide](https://github.com/huggingface/huggingface_hub/blob/main/docs/TRANSLATING.md) to get started 🤗

**Comment by HARSVARDH** [comment]
Hi, I Would like to translate the documentation into Telugu, Is this still available ? @Wauplin.

**Comment by Wauplin** [comment]
Yes @HARSVARDH !

**Comment by Raheel-glicth** [comment]
hi  @Wauplin  i want to translate documentation into my  state language
and see to cover north India in future

**Comment by Wauplin** [comment]
Yes sure @Raheel-glicth , go for it please? If your state language is not in the list, you can create a new issue from the template and let me know!

**Comment by la21unika** [comment]
Hi,

I had volunteered to translate into Spanish a couple years ago, but I
subsequently had to deal with some life changing events.  I would like to
do it now, but would like to know if you could share the link with me.

Thank you in advance for your help

Dru

**Comment by Wauplin** [comment]
@la21unika , Spanish issue is here! https://github.com/huggingface/huggingface_hub/issues/2640

**Comment by spideyashith** [comment]
Hi @Wauplin, I'd like to translate the documentation to Kannada. 
Is this still available? I'm a native Kannada speaker from Karnataka, India.

**Comment by Wauplin** [comment]
yes! commented on your PR 🤗

**Comment by indrajeetapache** [comment]
Hi! @Wauplin 

I would like to start an Odia translation effort for the huggingface_hub documentation.

I checked the existing translation issues but could not find one for Odia. The ISO 639-1 language code for Odia is or, so I understand that translations should be added under:

docs/source/or

This will be my first open-source contribution. I would like to begin with a small documentation page and follow the official TRANSLATING.md instructions.

**Comment by Wauplin** [comment]
Yes, please go ahead! Thanks for suggesting yourself 🤗

**Comment by Mr-Abhinav-Pandey** [comment]
Hi! I've opened a PR adding the Hindi translation for `guides/overview.md`.

PR: #4405

This contributes to the ongoing Hindi documentation effort in #1771. More pages can be translated incrementally in future PRs. 🙂

**Comment by indrajeetapache** [comment]
@Wauplin I have created one PR https://github.com/huggingface/huggingface_hub/pull/4454 can you please have a look ?

**Comment by kanaad-lims** [comment]
Hi! @Wauplin

I would like to start an Marathi translation effort for the huggingface_hub documentation.

I checked the existing translation issues but could not find one for Marathi. 

This will be my first open-source contribution. I would like to begin with a small documentation page and follow the official TRANSLATING.md instructions.

**Comment by indrajeetapache** [comment]
@Wauplin I have created one PR [454](https://github.com/huggingface/huggingface_hub/pull/4454) can you please have a look ?

**Comment by Wauplin** [comment]
@indrajeetapache I've merged https://github.com/huggingface/huggingface_hub/pull/4454, thank you!

@kanaad-lims , yes please go ahread 🤗

**Comment by kanaad-lims** [comment]
Hi! @Wauplin . I have opened the issue for the Marathi translation here: #4591 

For the Marathi translation I would be working on these first - 
- index.md
- quick-start.md
- installation.md

I'll open a PR once these translations are ready.

Thank you!

**Comment by kanaad-lims** [comment]
@Wauplin I have created a PR #4606 can you please take a look?
Thanks!

**Comment by prithamnandavaram** [comment]
@Wauplin Hi! I would like to contribute a Telugu translation of the Hugging Face Hub documentation. I’d be happy to work on this.

**Comment by nicolasdmolina** [comment]
Taking a small Spanish slice on #2640: `guides/overview.md` (How-to guides landing page). Get Started is already covered by #4401, so I will not duplicate those pages.

**Comment by rmouahid** [comment]
Hello, is it still avaliable for the french translation ?
